demucs/train.py

```diff
@@ -24,18 +25,40 @@
 from .hdemucs import HDemucs
 from .repitch import RepitchedWrapper
 from .solver import Solver
+from .states import capture_init
 
 logger = logging.getLogger(__name__)
 
 
+class TorchHDemucsWrapper(nn.Module):
+    """Wrapper around torchaudio HDemucs implementation to provide the proper metadata
+    for model evaluation.
+    See https://pytorch.org/audio/main/tutorials/hybrid_demucs_tutorial.html"""
```

Comment:
pointing this to the stable version of the tutorial

```suggestion
    See https://pytorch.org/audio/stable/tutorials/hybrid_demucs_tutorial.html"""
```